sig-architecture/production-readiness.md

```diff
@@ -41,6 +41,62 @@ aspects of the process.
   with and without the feature are necessary. At the very least, think about
   conversion tests if API types are being modified.
 
+
+#### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?**
+  Try to be as paranoid as possible - e.g. what if some components will restart
+  in the middle of rollout?
+
+* **What signals should trigger rollback?**
```

Comment:
This seems largely related to a given cluster-admins risk tolerance.  Can we rephrase to indicate what specific metrics inform a rollback?

Reply:
done